components/tooltip/README.md

```diff
@@ -163,15 +189,21 @@ The `d2l-tooltip-help` component is used to display additional information when
 <!-- docs: end donts -->
 <!-- docs: end best practices -->
 
-<!-- docs: demo live name:d2l-tooltip-help autoSize:false size:small -->
+<!-- docs: demo live name:d2l-tooltip-help autoSize:false size:medium -->
 ```html
 <script type="module">
-  import '@brightspace-ui/core/components/tooltip/tooltip-help.js';
+  import '@brightspace-ui/core/components/tooltip/tooltip-help.js'; <!-- docs: start hidden content --> window.addEventListener('load', function () {
+    setTimeout(function() {
+      var tooltip = document.querySelector('d2l-tooltip-help[text="Helpful label"]:not(#help-tooltip)').shadowRoot.querySelector('d2l-tooltip');
+      tooltip.showing = true;
+    }, 20);
+  });
+<!-- docs: end hidden content -->
 </script>
 
-<p class="d2l-body-standard">
+<p class="d2l-body-compact">
   This is some sample text.
-  <d2l-tooltip-help text="Helpful label" inherit-font-style>Contents should elaborate on the label (be short and concise)</d2l-tooltip-help>
```

Comment:
After discussing with Margaree and Jeff, we decided we didn't want this demo to override the default styling of the opener, even if the text next to it is styled differently